ksqldb-functional-tests/src/test/resources/query-validation-tests/joins.json

```diff
@@ -442,11 +442,14 @@
       "post": {
         "sources": [
           {"name": "OUTPUT", "type": "stream", "schema": "T_ID BIGINT KEY, NAME STRING, VALUE BIGINT, F1 STRING, F2 BIGINT"}
-        ]
+        ],
+        "topics": {
+          "blacklist": ".*-repartition"
+        }
       }
     },
     {
-      "name": "stream stream left join",
```

Comment:
The test immediately above this one has the same name. I've renamed this one to indicate the difference and avoid confusion.